server/resp/src/main/java/org/infinispan/server/resp/RespServer.java

```diff
@@ -85,6 +86,10 @@ protected void startInternal() {
             } else if (!keyMediaType.equals(RESP_KEY_MEDIA_TYPE)) {
                throw CONFIG.respCacheKeyMediaTypeSupplied(cacheName, keyMediaType);
             }
+
+            if (builder.transaction().lockingMode() != LockingMode.PESSIMISTIC)
+               org.infinispan.server.resp.logging.Log.CONFIG.utilizePessimisticLocking(builder.transaction().lockingMode().name());
+
          } else {
```

Comment:
I was thinking about that, but I think it would affect perf for users not using MULTI/EXEC operations. We already have a warning message suggesting enabling TX. Otherwise, users would need to define a configuration to disable TX. WDYT?

Reply:
If we have a warning somewhere else, that is fine.